SECURITY.md

```diff
@@ -1,32 +1,56 @@
+openness of Cosmos core contributors to help continually secure our operations.
 
-## Bug Bounty
+> **IMPORTANT**: *DO NOT* open public issues on this repository for security vulnerabilities.
 
-As part of our [Coordinated Vulnerability Disclosure Policy](https://tendermint.com/security), we operate a
-[bug bounty program](https://hackerone.com/cosmos) with Hacker One.
+## Scope
 
-See the policy linked above for more details on submissions and rewards and read
-this [blog post](https://blog.cosmos.network/bug-bounty-program-for-tendermint-cosmos-833c67693586) for the program scope.
+| Scope                 |
+|-----------------------|
+| last release (tagged) |
+| main branch           |
 
-The following is a list of examples of the kinds of bugs we're most interested
-in for the IBC Golang repository. Please refer to the corresponding repositories for vulnerabilities on the [Cosmos SDK](https://github.com/cosmos/cosmos-sdk/blob/master/SECURITY.md) and [CometBFT](https://github.com/cometbft/cometbft/blob/master/SECURITY.md) repositories.
+The latest **release tag** of this repository is supported for security updates as well as the **main** branch.
+Security vulnerabilities should be reported if the vulnerability can be reproduced on either one of those.
 
-### IBC Core
+## Reporting a vulnerability
 
-- [`02-client`](https://github.com/cosmos/ibc-go/tree/main/modules/core/02-client)
-- [`03-connection`](https://github.com/cosmos/ibc-go/tree/main/modules/core/03-connection)
-- [`04-channel`](https://github.com/cosmos/ibc-go/tree/main/modules/core/04-channel)
-- [`05-port`](https://github.com/cosmos/ibc-go/tree/main/modules/core/05-port)
-- [`23-commitment`](https://github.com/cosmos/ibc-go/tree/main/modules/core/23-commitment)
-- [`24-host`](https://github.com/cosmos/ibc-go/tree/main/modules/core/24-host)
+| Reporting methods                                                 |
+|-------------------------------------------------------------------|
+| [GitHub Private Vulnerability Reporting](security/advisories/new) |
+| [HackerOne bug bounty program](https://hackerone.com/cosmos)      |
 
-### IBC Applications
+All security vulnerabilities can be reported under GitHub's [Private vulnerability reporting](security/advisories/new) system.
+This will open a private issue for the developers. Try to fill in as much of the questions as possible. If you are not familiar
+with the CVSS system for assessing vulnerabilities, just use the Low/High/Critical severity ratings. A partially filled in report
+for a critical vulnerability is still better than no report at all.
 
-- [`transfer`](https://github.com/cosmos/ibc-go/tree/main/modules/apps/transfer)
+Vulnerabilities associated with the **Go or Protobuf code** of the repository may be eligible for a [bug bounty](https://hackerone.com/cosmos).
+Please see the bug bounty page for more details on submissions and rewards. If you think the vulnerability is eligible for a payout,
+**report on HackerOne first**.
```

Comment:
Is there a logistical reason why it would be difficult to reward folks if they open a private github issue instead of hackerone?

Reply:
No, I don't think so. But I guess we want to use hacker one as the main reporting interface. However, if we would like to give this option to users we need to enable [private vulnerability reporting](https://docs.github.com/en/code-security/security-advisories/repository-security-advisories/configuring-private-vulnerability-reporting-for-a-repository) first, actually.